source/blog/2016-05-30-dropping-dropbox.html.markdown

```diff
@@ -0,0 +1,99 @@
+
+<!-- more -->
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Things I don&#39;t want: <a href="https://twitter.com/hashtag/Dropbox?src=hash">#Dropbox</a> up in my kernel. <a href="https://t.co/p4LV87cKYy">pic.twitter.com/p4LV87cKYy</a></p>&mdash; Dusty (@duspom) <a href="https://twitter.com/duspom/status/735263193497505792">May 25, 2016</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+I've been trying to move away from online services that allow easy mass surveillance. For example, I started in 2012, when I [moved off of Gmail](https://ashfurrow.com/blog/switching-from-gmail-to-fastmail/). It's not been easy, I think in part because services that are popular are typically ad-supported – viewing your data is how they sell ads to you – this implies a lack of end-to-end encryption, and security in general. I've been tolerating Dropbox because it's so _damm_ convenient, but the kernel extension was the last straw, so I [asked for alternatives](https://twitter.com/ashfurrow/status/736936037876895744). 
+
+The solution that appealed to me the most was [BitTorrent Sync](https://getsync.com). It's closed source (boo) but offers a good compromise between security and convenience. I decided to give it a try, and in the process I ran into confusing documentation and advice that would make me _less_ secure than using Dropbox. It really sucks that in order to _securely_ do something as commonplace as syncing files, you have to be pretty tech savvy. While I'm continuing to evaluate it for a sync solution (I haven't moved off Dropbox yet), I'm hoping this post will be a resource for anyone who wants to use BitTorrent Sync securely.
+
+(But I'm hopeful that next month we'll see Apple show us some awesome end-to-end encryption features for iCloud Drive, too.)
+
+I am not a security expert, so I've marked the aspects of this post that I'm not 100% sure of with [?] so that I can be honest about what I know. If you can verify – or correct – something I'm unsure of, please [let me know](https://github.com/ashfurrow/blog/issues/new) or [send a correction](https://github.com/ashfurrow/blog/compare). My hope is that this post can evolve to be somewhat authoritative instructions on how to securely sync files across devices, aimed at people who are computer savvy but maybe not familiar with server administration and security (like me).
+
+## Overview
+
+Okay, so BitTorrent Sync is a service that runs on computers and synchronizes files across them using BitTorrent. Cool. Their site isn't clear on this, but you [need to buy a license](https://getsync.com/features#compare) to sync across multiple devices. It's $40/year, but has a 30-day trial (that they only tell you about after you download the app). Yeah, documentation needs some work.
+
```

Comment:
I don't follow then, because I couldn't "Add Device" without purchasing/starting a trial. I'll double-check the documentation, thanks!